crates/emmylua_code_analysis/resources/std/builtin.lua

```diff
@@ -125,3 +123,9 @@
 ---@alias collectgarbage_opt std.collectgarbage_opt
 
 ---@alias metatable std.metatable
+
+---@alias int integer
+
+---@alias str string
+
+---@alias bool boolean
```

Comment:
Note: I'm not sure about adding these. Personally, I'd rather had compact types under a feature flag, disabled by default.